editor/project_manager.h

```diff
@@ -93,42 +94,41 @@ class ProjectDialog : public ConfirmationDialog {
 
 	EditorFileDialog *fdialog = nullptr;
 	EditorFileDialog *fdialog_install = nullptr;
-	AcceptDialog *dialog_error = nullptr;
 
 	String zip_path;
 	String zip_title;
-	String fav_dir;
 
-	String created_folder_path;
+	void _set_message(const String &p_msg, MessageType p_type = MESSAGE_SUCCESS, InputType p_input_type = PROJECT_PATH);
+
+	String _validate_path();
 
-	void _set_message(const String &p_msg, MessageType p_type = MESSAGE_SUCCESS, InputType input_type = PROJECT_PATH);
+	String _last_target_auto_dir;
+	String _update_target_auto_dir(const String &p_path);
 
-	String _test_path();
-	void _path_text_changed(const String &p_path);
+	void _project_name_changed(const String &p_text);
+	void _project_path_changed(const String &p_text);
+	void _install_path_changed(const String &p_text);
 	void _path_selected(const String &p_path);
 	void _file_selected(const String &p_path);
 	void _install_path_selected(const String &p_path);
 
-	void _browse_path();
+	void _browse_project_path();
 	void _browse_install_path();
-	void _create_folder();
+	void _create_folder_toggled(const bool p_pressed);
```

Comment:
```suggestion
	void _create_folder_toggled(bool p_pressed);
```
Don't use `const` for primitive-type arguments.